docker/client.py

```diff
@@ -28,10 +28,14 @@
 from .auth import auth
 from .unixconn import unixconn
 from .ssladapter import ssladapter
-from .utils import utils, check_resource, update_headers
+from .utils import utils, check_resource, update_headers, kwargs_from_env
 from .tls import TLSConfig
 
 
+def from_env(**kwargs):
```

Comment:
I'm torn on the necessity of having this function in the module namespace when people could simply call `docker.Client.from_env()`